Description: move player_* to root dir 
 The player_* have to be move to the root dir to be executabel and comply with Debian guidlines.
 .
 ani-cli (3.0-1) UNRELEASED; urgency=medium
 .
   * Update to version 3.0
   * rewrite of the program
Author: Nils Wiemer <wiemernils@gmail.com>
Last-Update: 2022-06-15



---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: 2022-06-15

--- ani-cli-3.0.orig/ani-cli
+++ ani-cli-3.0/ani-cli
@@ -466,8 +466,8 @@ choice=
 auto_play=0
 project_root="$(dirname "$0")"
 case "$(uname)" in
-	Darwin*) . "$project_root/players/player_iina" || die "No iina player script found";; #MacOS specific
-	*) . "$project_root/players/player_mpv" || die "No mpv player script found";;
+	Darwin*) . "$project_root/player_iina" || die "No iina player script found";; #MacOS specific
+	*) . "$project_root/player_mpv" || die "No mpv player script found";;
 esac
 # history file path
 logfile="${XDG_CACHE_HOME:-$HOME/.cache}/ani-hsts"
@@ -480,24 +480,24 @@ logdir="${XDG_CACHE_HOME:-$HOME/.cache}"
 
 while getopts 'svq:dp:chDUVa:xf:' OPT; do
 	case $OPT in
-		d) . "$project_root/players/player_download" || die "No download player script found" ;;
+		d) . "$project_root/player_download" || die "No download player script found" ;;
 		a) ep_choice_to_start=$OPTARG ;;
 		D)
 			: > "$logfile"
 			exit 0
 			;;
 		p)
-			. "$project_root/players/player_download" || die "No download player script found"
+			. "$project_root/player_download" || die "No download player script found"
 			download_dir="$OPTARG"
 			;;
-		s) . "$project_root/players/player_syncplay" || die "No syncplay player script found";;
+		s) . "$project_root/player_syncplay" || die "No syncplay player script found";;
 		q) quality=$OPTARG ;;
 		x) debug=1 ;;
 		f) select_provider=$OPTARG ;;
 		c) scrape=History ;;
-		v) . "$project_root/players/player_vlc" || die "No vlc player script found";;
+		v) . "$project_root/player_vlc" || die "No vlc player script found";;
 		U)
-			update "$project_root/ani-cli" "$project_root/UI" "$project_root"/players/player_*
+			update "$project_root/ani-cli" "$project_root/UI" "$project_root"/player_*
 			exit 0
 			;;
 		V)
